ui/client/components/formikComponents/FormikSelect.js

```diff
@@ -3,9 +3,21 @@ import { useField, useFormikContext } from 'formik';
 
 import InputLabel from '@mui/material/InputLabel';
 import FormControl from '@mui/material/FormControl';
+import FormHelperText from '@mui/material/FormHelperText';
 import MenuItem from '@mui/material/MenuItem';
 import Select from '@mui/material/Select';
 
+// removes the [' '] from dataset multicolumn date & geo, which are there
```

Comment:
Great comment. I was trying to find evidence of this workaround online, and I did find an old github issue that illustrates this. Will leave here in the PR for reference:

https://github.com/jaredpalmer/formik/issues/2278